Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2704,43 +2853,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "114":
-    id: "114"
-    taskid: 18f94705-5309-4adb-8735-b0a2d4beaf66
+  "127":
+    id: "127"
+    taskid: d52809a1-410c-46f6-87ee-dca6c952bf3c
     type: regular
     task:
-      id: 18f94705-5309-4adb-8735-b0a2d4beaf66
+      id: d52809a1-410c-46f6-87ee-dca6c952bf3c
       version: -1
-      name: Close alert (remediated automatically) with rule match
-      description: Close the current alert because Automated Remediation path was selected.
-      script: Builtin|||closeInvestigation
+      name: Set playbook stage grid field (decision)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```